src/learner.Rmd

```diff
@@ -104,14 +104,14 @@ getHyperPars(cluster.lrn)
 getParamSet(classif.lrn)
 ```
 
-We can also use [&getParamSet] to get a quick overview about the available hyperparameters
+We can also use [&getParamSet] or its alias [&getLearnerParamSet] to get a quick overview about the available hyperparameters
 and defaults of a learning method without explicitly constructing it (by calling [&makeLearner]).
 
 ```{r}
 getParamSet("classif.randomForest")
 ```
 
-Functions for accessing Learner's meta information are available in [%mlr]. We can use ``getLearnerId``,``getLearnerShortName`` and  ``getLearnerType`` to get Learner's id, short name and class, respectively. Moreover, in order to show the required packages for the Learner, one can call ``getLearnerPackages``.
+Functions for accessing a Learner's meta information are available in [%mlr]. We can use [%getLearnerId], [%getLearnerShortName] and  [%getLearnerType] to get Learner's id, short name and type, respectively. Moreover, in order to show the required packages for the Learner, one can call [%getLearnerPackages].
```

Comment:
You need & instead of %.
(& is for links to function documentation and % for links to packages)